config_setup_example.py

```diff
@@ -0,0 +1,5 @@
+## SET YOUR VALUES IN A COPY OF THIS CALLED config_setup.py
```

Comment:
Is there any reason why you have 2 config files instead of 1?

Reply:
Separating animation setup and overall config makes sense in my opinion

Reply:
Like Artem says, yeah, one is an overall config (one off) and an animation config (one par animation). The idea is that you just as you make each animation make tons of copies of the animation config one and whichever one is named config_ani is the config that will be used.